MAUI/Scheduler/cell-selection.md

```diff
@@ -76,7 +76,7 @@ this.Scheduler.CellSelectionView.CornerRadius = 2;
 
 The thickness of the `Stroke` of Scheduler can be customized by using the `StrokeWidth` property in the `CellSelectionView`.  
```

Comment:
The thickness of the selection view `Stroke` can be customized by using the `StrokeWidth` property in the `CellSelectionView`.

Reply:
Updated